.github/workflows/release.yml

```diff
@@ -14,3 +14,15 @@ jobs:
             -H "Content-Type: application/json" \
             --data '{"event_type": "release", "client_payload": { "repo": "deno-drash", "module": "drash", "version": "${{ github.ref }}" }}' \
             https://api.github.com/repos/drashland/castle/dispatches
+
+  update-middleware-repo:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify Middleware about this release
+        run: |
+          curl -X POST \
+            -u "${{ secrets.CI_USER_NAME }}:${{ secrets.CI_USER_PAT }}" \
+            -H "Accept: application/vnd.github.everest-preview+json" \
+            -H "Content-Type: application/json" \
+            --data '{"event_type": "release", "client_payload": { "repo": "deno-drash", "module": "drash" }}' \
+            https://api.github.com/repos/drashland/deno-drash-middleware/dispatches
```

Comment:
we need to catch this event in middleware right?

Reply:
Yeah that’s gonna be done as a separate PR